apps/web/src/common/db.ts

```diff
@@ -39,14 +39,33 @@ async function initializeDatabase(persistence: DatabasePersistence) {
     await useKeyStore.getState().setValue("databaseKey", databaseKey);
   }
 
+  // Get server URLs from environment variables (with NN_ prefix) or user config
+  const getServerUrl = (host: string, defaultValue: string): string => {
+    // Check for environment variable with NN_ prefix first
+    const envVar = `NN_${host}`;
+    const envValue = (import.meta as any).env[envVar];
+    if (envValue) {
+      return envValue;
+    }
+    
+    // Fall back to user config
```

Comment:
You don't need to add these unnecessary comments.

Reply:
Honestly, I'm mainly a python developer and just trying to get the ball rolling.  Thank you for your guidance.  I'll make the changes in the AM (GMT-8 here in Alaska)